gcsfs/core.py

```diff
@@ -488,6 +491,25 @@ def _call(self, method, path, *args, **kwargs):
             out = r.content
         return out
 
+    @_tracemethod
+    def _post(self, path, **kwargs):
+
+        for retry in range(self.retries):
```

Comment:
May be best to factor out the retry logic "call to be retries" and "number of retries" to be a function unto itself, and if it runs out of retries, it raises the last exception.

Reply:
I understand and it does make sense to consider using a generic `_call` but this is beyond my expertise so please do all edits you want, or if someone else can take over. 
To add on my previous comment that `post` requests need more than the response json, `_fetch_range` could also benefit from retries but it uses `raise_for_status()`.

To me any request should be retried if error is transient hence retriable.